kyuubi-common/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>hive-service-rpc</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
who/where use this in the common module？

Reply:
We don't use it. `jaxb-api` should be considered as the transitive deps of `hadoop-client-runtime`. 
If we move `jaxb-api` to test scope, and accident invocation of Hadoop API which direct or indirect requires `jaxb-api`, the ut pass, but will cause runtime failures in production.